website/reference.py

```diff
@@ -276,6 +276,32 @@ def update():
         ui.button('Update', on_click=update)
         ui.button('Select all', on_click=lambda: table.call_api_method('selectAll'))
 
+    @example(ui.qtable, menu)
+    def qtable_example():
+        fields = [
+            {'name': 'name', 'field': 'name', 'label': 'Name', 'required': True},
+            {'name': 'age', 'label': 'Age', 'field': 'row => row.age.deep', 'sortable': True},
+        ]
+
+        data = [
+            {'name': 'Alice', 'age': {'deep': 18}},
+            {'name': 'Bob', 'age': {'deep': 21}},
+            {'name': 'Carol', 'age': {'deep': 42}},
+        ]
+
+        table = ui.qtable(
+            columns=fields,
+            rows=data,
+            selection_mode='single',
+            selection_key='name',
+            title='QTable',
+            filter_config={'label': 'Search...', 'icon': 'search', 'color': 'primary', 'dense': True})
```

Comment:
See my v-if suggestion above and the B0002 comment to have it to None by default to remove the search bar at all.